scenario/consistency_checker.py

```diff
@@ -584,6 +585,11 @@ def check_containers_consistency(
                 f"the event being processed concerns notice {event.notice!r}, but that "
                 "notice is not in any of the containers present in the state.",
             )
+        if event.check_info and event.check_info.name not in all_checks:
```

Comment:
Should a check be keyed to a (container_name, check_name) tuple?
Otherwise, validation here would allow correct check name for a wrong container.

Reply:
Nice catch, thanks! I should probably add an E2E test for this, two containers both having the same check name.